lib/erl/src/thrift_compact_protocol.erl

```diff
@@ -355,7 +356,7 @@ read(This0, i64) ->
 read(This0, double) ->
   {This1, Bytes} = read_data(This0, 8),
   case Bytes of
-    {ok, <<Val:64/float-signed-big, _/binary>>} -> {This1, {ok, Val}};
+    {ok, <<Val:64/float-signed-little, _/binary>>} -> {This1, {ok, Val}};
```

Comment:
Ooops, the `Double` part is just a variable, the same as `Val`.
Anyway I'll unify the order.